Tests/BackendIntegrationTests/BaseBackendIntegrationTests.swift

```diff
@@ -34,6 +34,8 @@ class BaseBackendIntegrationTests: XCTestCase {
     // swiftlint:disable:next weak_delegate
     private(set) var purchasesDelegate: TestPurchaseDelegate!
 
+    private var mainThreadMonitor: MainThreadMonitor!
```

Comment:
This is a cool idea. Any thoughts on using this in other integration tests (not only backend integration tests)?

Reply:
This is the base class for all integration tests.